src/app/components/layout/navbar/navbar.component.html

```diff
@@ -336,6 +336,29 @@
         </div>
       </div>
     </div>
+    <div class="navbar-item">
+      <div *ngIf="pingInfo.isAlive && showBellIcon && getUpdateInterval() !== '-1'">
+        <div class="dropdown is-right is-hoverable">
+          <div class="dropdown-trigger">
+            <a aria-haspopup="true" aria-controls="dropdown-menu" title="Updates">
+              <span class="icon">
+                <i class="fa-regular fa-bell" aria-hidden="true"></i>
+              </span>
+            </a>
+          </div>
+          <div class="dropdown-menu" id="dropdown-menu" role="menu">
+            <div class="dropdown-content">
+              <a class="dropdown-item update-dropdown-item">
+                <span>A newer version is available.
+                  <!-- TODO: It is commented till the polling of the update call based on ping alive after update -->
```

Comment:
You mean polling based on ping response is received after restart & check update call returns []